media-libs/openexr/Manifest

```diff
@@ -2,3 +2,4 @@ DIST openexr-1.7.0.tar.gz 13542910 BLAKE2B f187aa1b7c703bc912bfc87eb6f9d44f1ed08
 DIST openexr-2.0.1.tar.gz 12609335 BLAKE2B 25adcf5a679a0419a7ff0ddd29d4d1c5f1c28cd3fd6c81b0a5c0cc63d907e741eaea378f4ebaa9a66afce0349edf7231ebd2bb3e493aee484860c89f05f45960 SHA512 b35e3e795496f9d4a56352c907bcffef14ae062b189c4e1df35f3ab08c5e9abf829fa9149db3198d60088fa9633816650011e5f7871e07324801b95549d6695b
```

Comment:
Commit message:
1. Please use a single footer, and separate footer tags from body.
2. Please use `Closes` consistently.
3. I have no clue what that `~...` line at the end is supposed to mean.